packages/firestore/src/model/values.ts

```diff
@@ -578,9 +559,23 @@ export function refValue(databaseId: DatabaseId, key: DocumentKey): api.Value {
   };
 }
 
+/** Returns true if `value` is either an IntegerValue . */
+export function isInteger(
+  value?: api.Value | null
+): value is { integerValue: string | number } {
+  return !!value && 'integerValue' in value;
+}
+
+/** Returns true if `value` is either an IntegerValue . */
+export function isDobule(
```

Comment:
s/isDobule/isDouble/

Also, comment is out of date.

Reply:
Fixed (also renamed contains to arrayValueContains)